src/remoted/ar-forward.c

```diff
@@ -37,11 +37,9 @@ void *AR_Forward(__attribute__((unused)) void *arg)
         merror_exit(QUEUE_ERROR, path, strerror(errno));
     }
 
-    memset(msg, '\0', OS_SIZE_1024 + 1);
-
     /* Daemon loop */
     while (1) {
-        if (OS_RecvUnix(arq, OS_SIZE_1024, msg)) {
+        if (OS_RecvUnix(arq, OS_MAXSTR, msg)) {
```

Comment:
I think we should give `OS_MAXSTR - 1` so we prevent `OS_RecvUnix` from overflowing `msg`.
`OS_RecvUnix(buf, n)` writes into `buf[n]`.

I propose:
```c
OS_RecvUnix(arq, OS_MAXSTR - 1, msg);
```